src/additional_styles/button.css

```diff
@@ -0,0 +1,23 @@
+.button {
+  background-color: aquamarine;
+  border: 1px solid #222;
+  border-radius: 10px;
+  box-sizing: border-box;
+  color: #222;
+  cursor: pointer;
+  display: inline-block;
+  font-family: Circular, -apple-system, Roboto, "Helvetica Neue", sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 10px;
+  margin: 0;
+  outline: none;
+  padding: 13px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  touch-action: manipulation;
+  user-select: none;
+  -webkit-user-select: none;
+  width: auto;
+}
```

Comment:
I would recommend splitting rules into small groups according to their purpose (e.g. positioning, formatting etc.) - that's gonna make it much more readable😃

Reply:
it's just ready & copied block of code 😁, okay)